go.mod

```diff
@@ -83,3 +83,10 @@ replace (
 	github.com/openshift/machine-config-operator => github.com/openshift/machine-config-operator v0.0.0-20200123151440-ca3e3e1921f3 // release-4.4
 	golang.org/x/tools => golang.org/x/tools v0.0.0-20191206213732-070c9d21b343
 )
+
+// Test deps
+replace (
+	github.com/openshift-kni/performance-addon-operators => github.com/openshift-kni/performance-addon-operators v0.0.0-20200514122016-65bb21a3eb94 // master
```

Comment:
@fedepaol  why pin against master and not against release-4.5 ?  (https://github.com/openshift-kni/performance-addon-operators/commits/release-4.5)

Reply:
It's because ptp / sriov are not cutting the 4.6 branch. This master should point to 4.6. I left it explicit so it's easier to notice.

Reply:
@fedepaol  just FYI, currently I see 3 failures around performance addons that related to initramfs image but it was removed completely in https://github.com/openshift-kni/performance-addon-operators/pull/190/  (which is a newer commit in performance addons)

Reply:
Ok, will update the pr